startform/src/test/kotlin/nl/nlportal/startform/StartFormTestConfiguration.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.nlportal.startform
+
+import nl.nlportal.core.security.OauthSecurityAutoConfiguration
+import org.apache.commons.lang3.StringUtils
+import org.springframework.beans.factory.ObjectProvider
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.SpringBootConfiguration
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.boot.autoconfigure.security.SecurityProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.security.core.userdetails.MapReactiveUserDetailsService
+import org.springframework.security.core.userdetails.User
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.security.crypto.password.PasswordEncoder
+import java.util.regex.Pattern
+
+@SpringBootConfiguration
+@EnableAutoConfiguration(exclude = [OauthSecurityAutoConfiguration::class])
+class StartFormTestConfiguration {
```

Comment:
Waarom gaan we bij deze TestConfiguration opeens basic authentication gebruiken. Volgens mij wordt het in andere modules op een andere manier opgelost

Reply:
In veel modules word dit wel gebruikt